### ISSUE TITLE
[BUG] posts stuck in bookmarks

### ISSUE THREAD

**Issue by Dawn11111111** [vitorpamplona/amethyst#1100]
**Describe the bug**
Some long-form posts cannot be removed from private bookmarks. The "remove" option changes to the "add" option, but the note stays.
I'm note sure which clients are used to publish, or which event type these technically are. 

**To Reproduce**
1. Here is an example of one that won't leave: nostr:naddr1qqv9xatsv4exjmn5v4kxc6t8v4hxxefdd43xvanrx5pzq6knu235sx9320yp7jx93azw2xv70d8u3klr0qg2qqxuu0yska6qqvzqqqr4gu4lmnda
2. I do have a couple others if you need more.

**Expected behavior**
Note should remove as anticipated.

**Video and Screenshots**
![Screenshot_20240925_145745_Amethyst](https://github.com/user-attachments/assets/cbaa8b89-847e-4c5e-9461-03d163842132)

**Device (please complete the following information):**
 - Phone Brand/Model [e.g. Pixel 7 Pro]: Galaxy 20 5g
 - Android Version [e.g. 33]: 13
 - App Version: 91.0-PLAY

 - Amber Version (if using it to sign):

**Bounty (in Bitcoin sats) offered for a solution**
Incentivize developers to work on your issue. Describe clear milestones to claim payment.

**Comment by Dawn11111111** [comment]
There's another bookmark related issue. If a note is saved as both private & public bookmarks, then removed from private, it gets removed from both places. 

The inverse does not happen. I didn't know if I should make this a separate bug report at the moment. I imagine you're all busy enough for the time being :)

**Comment by davotoula** [comment]
I wasn't able to reproduce the top issue. Can you try again please?

**Comment by Dawn11111111** [comment]
I'm not sure how to tell which long-form posts will or will not do it, so i'm not sure how to reproduce it on purpose. Here is a more recent note id that i currently have stuck in Amethyst private bookmarks, if that helps. I just left a question for the author, as to which client was used to publish it. If I get a response, I will let you know.
 nostr:naddr1qq24jstj2prng729dpc9st2xtpuyg4mgxqmxjq3qwkrfe7kyksfn8s4zmnkr9c9ueakc8hz2fhu4mdayc0gfxa5pnxrsxpqqqp65wk0pvsn

**Comment by davotoula** [comment]
Well, both examples you've listed work fine for me.

I can add and remove multiple times.

I can add to both private and public and remove individually.

I'm starting to suspect your local state. Maybe there was a bug in an earlier version so if you used it then it messed up your local settings. 

Would it be too much trouble to uninstall and reinstall?

Of course you have to have your nsec backed up. And you will lose some settings like emojis, zap amounts and nwc if you have configured it.

Let me know if you are up for it.

**Comment by Dawn11111111** [comment]
Makes sense. I can do that. It will be a couple days before I can sift through & make note of the things I don't want to lose,  but I'll let you know how it goes. Thanks!

**Comment by Dawn11111111** [comment]
That seems to have worked, thank you
